### PR TITLE
Person Portfolio: Desktop version completed.

### PR DESCRIPTION
**Pull request by SamTush**
The complete version of the desktop version.

- develop desktop version to be almost the same as Figma

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -123,15 +135,17 @@ <h4>CANOPY</h4>
...
             </div>
-            <div class="grid-item grid-item-4">
+            <div  id="reverse" class="grid-item grid-item-4">
                 <div class="card-box">
-                    <img src="images/Snapshoot Portfolio4.png" alt="" srcset="">
+                    <img class="mobile-image" src="images/Snapshoot Portfolio4.png" alt="" srcset="">
+                    <img class="desktop-image" src="images/Snapshoot-Portfolio5.png" alt="" srcset="">
                 </div>
                 <div class="text-box">
                     <h2>Multi-Post<br>Stories</h2>
```
Comment (Gambit142) [inline index.html]: - [ ] Kindly follow the design template when designing the header of the cards with the title **_"Multi-Post Stories"_**

### Your Design

image

### Figma Design

image

File: index.html [hunk 2]
```diff
@@ -216,7 +231,7 @@ <h3 class="connect">
         <div class="contact">
             <div class="text-contact">
                 <h2>Contact me</h2>
-                <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
+                <p class="footerp">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it
                 </p>
                 <div class="form-div">
                     <form action="https://formspree.io/f/xvongqly"
```
Comment (Gambit142) [inline index.html:239]: - [ ] According to the requirement, the form button should be aligned to the middle on the desktop version but yours is aligned to the left of the form. Kindly implement this requirement. 👍 

### Your Design

image

### Figma Design

image

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @SamTush 👋, 

Good job so far!

## Highlights
- All linters checks passed ✔️ 
- README.md file is professional ✔️ 
- Used media query between `768` - `1440` px

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:

N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please, remember to tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by huseyinbiyik — APPROVED** [review]
Hi @SamTush,

## ✔️ Status: APPROVED 🏆 🎉 🎊

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Alt Text



## Highlights
- Linter set up well and checks pass 🟢
- Github flow followed 👌
- Commit messages are meaningful and written in imperative tense 📝
- Great implementation of the requested changes of previous reviewers 🕸️

## Optional suggestions
[OPTIONAL] Good job with aligning your contact form submit button on desktop. Please make sure your mobile version matches the Figma template design.
Screenshot 2023-01-24 at 10 28 54 PM
Screenshot 2023-01-24 at 10 29 55 PM

[OPTIONAL] Please make sure your website is responding to different sizes. When you change the screen width there are some parts breaking, nav is not extending etc. Check the gif below.
Screen Recording 2023-01-24 at 10 19 41 PM

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**